src/modules/init/buttons.ts

```diff
@@ -33,7 +33,17 @@ const getButton = (namespace: string, {
   buttonEl.classList.add(btnNamespaceClass);
 
   if (className) {
-    buttonEl.classList.add(className);
+    if (typeof className === 'string') {
+      className.split(' ')
+        .filter(name => name.length > 0)
+        .forEach(name => {
+          buttonEl.classList.add(name);
+        });
+    } else if (Array.isArray(className)) {
+      className.forEach(name => {
```

Comment:
Please check the length of the name here, and also DRY when adding the class names. In both cases, we get an array from the input, so you could instead do this
```
const classNameArray = Array.isArray(className)? className: className.split(' ')
classNameArray.forEach(name => {
buttonEl.classList.add(name);
});
```
Makes sense?

Reply:
Sure, great idea.
Do you agree with this?
```js
const classNameArray = Array.isArray(className) ? className : className.split(' ');
classNameArray
  .filter(name => name.length > 0)
  .forEach(name => {
    buttonEl.classList.add(name);
  });
```

Reply:
The first line can be split into parts for clarity if you want but yes, that's the idea.